app/models/art_item.rb

```diff
@@ -46,23 +47,38 @@ def has_description
       errors.add(:description, "Can't be blank")
     end
   end
+ 
+  private
 
-  def acceptable_documents
+  def validate_documents
     return unless documents.attached?
-  
-    acceptable_types = [
-      "application/pdf", "text/plain", "image/jpg", 
-      "image/jpeg", "image/png", 
-      "application/vnd.openxmlformats-officedocument.wordprocessingml.document"
-    ]
 
-    documents.each do |doc|
-      unless doc.byte_size <= 20.megabyte
-        errors.add(:documents, "is too big")
+    acceptable_doc_types = ['application/pdf', 'application/msword', 
+                          'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 
+                          'text/plain', 'application/rtf', 'application/vnd.ms-excel', 
+                          'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet']
```

Comment:
Since we have word and excel in the list, how about also allowing powerpoint documents ?

Reply:
I guess powerpoint seems not really useful in this context